app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -2,8 +2,35 @@ class OmniauthCallbacksController < ApplicationController
   def callback
     auth = request.env["omniauth.auth"]
 
-    session[:user_info] = auth.extra.raw_info
+    session[:user_info] = if Rails.env.development?
+      {
+        "trn" => "1886088",
+        "family_name" => "John",
+        "given_name" => "Smith",
+        "birthdate" => "1990-11-09",
+        "ni_number" => "TK519024B",
+        "trn_match_ni_number" => "true",
+        "email" => "john.smith@example.com",
+        "phone_number" => "01234567890"
+      }
+    else
+      auth.extra.raw_info
+    end
```

Comment:
I think we would want to be able to use the pre-production Teacher ID environment in development as well for testing. Perhaps this could be done with an optional configuration?

Reply:
This was meant to be kept locally 🤦  Removing now.
On that note a feature flag in local to switch between this and preprod could be nice.

Reply:
Removed